py/circuitpy_mpconfig.mk

```diff
@@ -241,6 +241,10 @@ CFLAGS += -DCIRCUITPY_GNSS=$(CIRCUITPY_GNSS)
 CIRCUITPY_I2CPERIPHERAL ?= $(CIRCUITPY_FULL_BUILD)
 CFLAGS += -DCIRCUITPY_I2CPERIPHERAL=$(CIRCUITPY_I2CPERIPHERAL)
 
+# Support for use of internal pullups with I2C
+CIRCUITPY_I2C_INTERNAL_PULLUPS ?= 0
+CFLAGS += -DCIRCUITPY_I2C_INTERNAL_PULLUPS=$(CIRCUITPY_I2C_INTERNAL_PULLUPS)
+
```

Comment:
Rename to `CIRCUITPY_BOARD_I2C_USE_INTERNAL_PULLUPS`, since that is now more accurante.